clients/src/main/java/org/apache/kafka/common/security/ssl/SSLFactory.java

```diff
@@ -60,12 +60,13 @@ public void configure(Map<String, ?> configs) throws KafkaException {
 
         if (configs.get(SSLConfigs.SSL_CIPHER_SUITES_CONFIG) != null) {
             List<String> cipherSuitesList = (List<String>) configs.get(SSLConfigs.SSL_CIPHER_SUITES_CONFIG);
-            this.cipherSuites = (String[]) cipherSuitesList.toArray(new String[cipherSuitesList.size()]);
+            if (!cipherSuitesList.isEmpty())
+                this.cipherSuites = cipherSuitesList.toArray(new String[cipherSuitesList.size()]);
```

Comment:
Out of curiosity, why are we doing the `isEmpty` check here? We don't do the same check when setting the enabledProtocols. It seems to add noise for little benefit given how this is used.